aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java

```diff
@@ -509,6 +538,26 @@ public AwsProperties(Map<String, String> properties) {
             properties,
             S3FILEIO_MULTIPART_UPLOAD_THREADS,
             Runtime.getRuntime().availableProcessors());
+    this.s3PathStyleAccess =
+        PropertyUtil.propertyAsBoolean(
+            properties,
+            AwsProperties.S3FILEIO_PATH_STYLE_ACCESS,
+            AwsProperties.S3FILEIO_PATH_STYLE_ACCESS_DEFAULT);
+    this.s3UseArnRegionEnabled =
+        PropertyUtil.propertyAsBoolean(
+            properties,
+            AwsProperties.S3_USE_ARN_REGION_ENABLED,
```

Comment:
[question] do we need to `AwsProperties.S3_USE_ARN_REGION_ENABLED` , ` S3_USE_ARN_REGION_ENABLED` should be suff,  as we are already in AwsProperties (copy to all places) ?

Reply:
Ah, that's my mistake. I forgot to delete the `AwsProperties.` after copy-pasting the code. Thank you for pointing it out. 

I will also change other `AwsProperties.[FLAG_NAME]` to `[FLAG_NAME]` to make the `AwsProperties` constructor cleaner.